examples/moondream-chatbot/bot.py

```diff
@@ -68,7 +68,7 @@
 
 class TalkingAnimation(FrameProcessor):
     """This class starts a talking animation when it receives an first AudioFrame,
```

Comment:
Shouldn't this be `BotStartedSpeakingFrame`?

```suggestion
    """This class starts a talking animation when it receives an first BotStartedSpeakingFrame,
```